src/Language/Fixpoint/Names.hs

```diff
@@ -20,20 +19,47 @@ module Language.Fixpoint.Names (
   -- * Symbols
     Symbol
   , Symbolic (..)
-  , anfPrefix, tempPrefix, vv, isPrefixOfSym, isSuffixOfSym, stripParensSym
-  , consSym, unconsSym, dropSym, singletonSym, headSym, takeWhileSym, lengthSym
-  , symChars, isNonSymbol, nonSymbol
+
+  -- * Conversion to/from Text
+  , symbolSafeText
+  , symbolSafeString
+  , symbolText
+  , symbolString
+
+  -- Predicates
+  , isPrefixOfSym
+  , isSuffixOfSym
+  , isNonSymbol
   , isNontrivialVV
-  , symbolText, symbolString
-  , encode, vvCon
-  , dropModuleNames
-  , dropModuleUnique
-  , takeModuleNames
+
+  -- * Destructors
+  -- , stripParensSym
+  -- , stripParens
+  , stripPrefix
+  , consSym
+  , unconsSym
+  , dropSym
```

Comment:
I plan to remove most of these functions on `Symbol`, basically anything of type `a -> Symbol` other than `symbol` itself. Constructing a `Symbol` is expensive and we should discourage manipulating `Symbol`s (as opposed to manipulating a `Text` and converting it to a `Symbol` at the end).